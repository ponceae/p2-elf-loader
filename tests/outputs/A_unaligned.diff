--- outputs/A_unaligned.txt	2021-11-15 14:37:49.064959074 -0500
+++ expected/A_unaligned.txt	2021-10-08 11:04:23.000000000 -0400
@@ -10,6 +10,8 @@
   02       0x0057    0x013a    0x0008    DATA      RW 
 Contents of memory from 0000 to 0009:
   0000  70 18 01 00 00 00 00 00  00
-Contents of memory from 0118 to 011a:  10 00
-Contents of memory from 013a to 0142: 39 30 00 00 00 00
+Contents of memory from 0118 to 011a:
+  0110                           10 00
+Contents of memory from 013a to 0142:
+  0130                                 39 30 00 00 00 00
   0140  00 00
