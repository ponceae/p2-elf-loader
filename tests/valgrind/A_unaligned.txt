==298648== Memcheck, a memory error detector
==298648== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==298648== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==298648== Command: ../y86 -a inputs/unaligned.o
==298648== 
01 00 18 01 10 00 03 00  00 00 00 00 45 4c 46 00
Mini-ELF version 1
Entry point 0x118
There are 3 program headers, starting at offset 16 (0x10)
There is no symbol table present
There is no string table present
 Segment   Offset    VirtAddr  FileSize  Type      Flag
  00       0x004c    0x0000    0x0009    CODE      R X
  01       0x0055    0x0118    0x0002    CODE      R X
  02       0x0057    0x013a    0x0008    DATA      RW 
Contents of memory from 0000 to 0009:
  0000  70 18 01 00 00 00 00 00  00
Contents of memory from 0118 to 011a:  10 00
Contents of memory from 013a to 0142: 39 30 00 00 00 00
  0140  00 00
==298648== 
==298648== HEAP SUMMARY:
==298648==     in use at exit: 0 bytes in 0 blocks
==298648==   total heap usage: 4 allocs, 4 frees, 20,952 bytes allocated
==298648== 
==298648== All heap blocks were freed -- no leaks are possible
==298648== 
==298648== For lists of detected and suppressed errors, rerun with: -s
==298648== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
