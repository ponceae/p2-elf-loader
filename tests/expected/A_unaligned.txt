01 00 18 01 10 00 03 00  00 00 00 00 45 4c 46 00
Mini-ELF version 1
Entry point 0x118
There are 3 program headers, starting at offset 16 (0x10)
There is no symbol table present
There is no string table present
 Segment   Offset    VirtAddr  FileSize  Type      Flag
  00       0x004c    0x0000    0x0009    CODE      R X
  01       0x0055    0x0118    0x0002    CODE      R X
  02       0x0057    0x013a    0x0008    DATA      RW 
Contents of memory from 0000 to 0009:
  0000  70 18 01 00 00 00 00 00  00
Contents of memory from 0118 to 011a:
  0110                           10 00
Contents of memory from 013a to 0142:
  0130                                 39 30 00 00 00 00
  0140  00 00
